test/e2e/fixtures/given.go

```diff
@@ -172,9 +172,6 @@ func (g *Given) Exec(name string, args []string, block func(t *testing.T, output
 	g.t.Helper()
 	output, err := Exec(name, args...)
 	block(g.t, output, err)
-	if g.t.Failed() {
```

Comment:
`FailNow` (and therefore `Fatal`) cause both the current test, and the whole suite, to stop running. This means only one test can fail per suite, but there maybe more you want to know about.